sync/sync.go

```diff
@@ -196,6 +203,10 @@ func (t *Task) AddReplica(replicaAddress string, s *replica.Server) error {
 	if s == nil {
 		return fmt.Errorf("Server not present for %v, Add replica using CLI not supported", replicaAddress)
 	}
+	logrus.Infof("CheckAndResetFailedRebuild %v", replicaAddress)
```

Comment:
do we have a testcase for this change? existing testcases may cover this as well if I'm not wrong

Reply:
this is another good one

Reply:
I thought this is good one as 'rebuilding' status will be set to false, but, the function consists of a condition along with 'state'.. and, there seems to be a Open call inside this.. this is going to take time..
so, why we need this change?

Reply:
I mean - it should be because we removed 'start' action when rebuild is set.. but, there is condition along with 'state' to be closed.. will that be the case always?
And, having 'Open' call in this is going to impact time taken for first replica to become connected and RW?

Reply:
Added setting preload to false in this. And also removed the extra condition which will always be true.